.github/workflows/build-ot3-actions.yml

```diff
@@ -5,30 +5,33 @@ on:
     inputs:
       monorepo-ref:
         description: |
-          Ref of https://github.com/opentrons/opentrons to build. This MUST be a full ref, e.g. refs/heads/edge, or '-' to indicate not-specified. If not specified, will be determined from the oe-core ref if specified, and then default to edge.
+          Ref of https://github.com/opentrons/opentrons to build. ONLY use a tag ref if you are trying to actually release!!! This MUST be a full ref, e.g. refs/heads/edge or refs/tags/v7.5.0-alpha.1 or '-' to indicate not-specified. If not specified, will be determined from the oe-core ref if specified, and then default to edge.
         required: true
         default: '-'
       oe-core-ref:
         description: |
-          Ref of https://github.com/opentrons/oe-core to build. This is different from the ref specified in the github api/webUI when starting this workflow - that ref is what contains this workflow, this ref specifies what gets built. It MUST be a full ref, e.g. refs/heads/main, or '-' to indicate not-specified. If not specified, will be decided based on the monorepo ref; if that isn't specified, will be main.
+          Ref of https://github.com/opentrons/oe-core to build. This is different from the ref specified in the github api/webUI when starting this workflow - that ref is what contains this workflow, this ref specifies what gets built. ONLY use a tag ref if you are trying to actually release!!! It MUST be a full ref, e.g. refs/heads/main or refs/tags/v0.6.4 or '-' to indicate not-specified. If not specified, will be decided based on the monorepo ref; if that isn't specified, will be main.
         required: true
         default: '-'
       ot3-firmware-ref:
         description: |
-          Ref of https://github.com/opentrons/ot3-firmware to build. It MUST be a full ref, e.g. refs/heads/main, or '-' to indicate not-specified. If not specified, will be decided based on the monorepo ref; if that isn't specified, will be main.
+          Ref of https://github.com/opentrons/ot3-firmware to build. ONLY use a tag ref if you are trying to actually release!!! It MUST be a full ref, e.g. refs/heads/main or refs/tags/v52 or '-' to indicate not-specified. If not specified, will be decided based on the monorepo ref; if that isn't specified, will be main.
         required: false
         default: '-'
       infra-stage:
         description: |
-          What infra stage to run on. This should almost always be prod; staging and dev are useful when you explicitly want to test those infra stages.
+          What infra stage to run on. This should almost always be prod; dev is useful when you explicitly want to test or prod is busy.
         required: true
         type: choice
         options:
           - 'stage-prod'
-          - 'stage-staging'
           - 'stage-dev'
         default: 'stage-prod'
 
+concurrency:
```

Comment:
We have concurrency here

https://github.com/Opentrons/oe-core/blob/8aa4de42a68dde3a9630cdc2bb0ef4d4d3928ea4/.github/workflows/build-ot3-actions.yml#L67

Reply:
Just saw that, fixing